setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = df_to_azure
-version = 0.6.0
```

Comment:
I think we have to bump this to version 0.7.0. Since potentially there can be changes in functionality with upgrading the packages.